trackintel/io/file.py

```diff
@@ -74,14 +74,14 @@ def read_positionfixes_csv(*args, **kwargs):
     >>> trackintel.read_positionfixes_csv('data.csv')
     >>> trackintel.read_positionfixes_csv('data.csv', columns={'time':'tracked_at', 'User':'user_id'})
     """
-    columns = kwargs.pop('columns', {})
-    tz = kwargs.pop('tz', None)
-    crs = kwargs.pop('crs', None)
-    
+    columns = {} if columns is None else columns
+
     # Warning if no 'index_col' parameter is provided
-    if 'index_col' not in kwargs:
+    if index_col is None:
```

Comment:
I think the implementation now has a problem because there are some cases where you can not get rid of the warning.
The problem is that there are cases where there is no index in the .csv file and a new index should be generated and cases where there is an index in the .csv file but the user forgot to define the column. Our Idea now was to check if the argument `index_col` was set explicitly to None by the user. In a way it was like if there are two different 'None' cases. 

There is a discussion about this on [stackoverflow](https://stackoverflow.com/questions/14749328/how-to-check-whether-optional-function-parameter-is-set) but most of the solutions are rather wild...
I think the best option would be [this](https://stackoverflow.com/a/14749388) where the default value is something that the user would never pick. E.g., `index_col=object()` and then check if the parameter is an object and set it to None afterwards. 
If we do it like this, then the warning could stay the same as it would be prevented if `index_col` is set to None explicitly.